docs/modules/deployment/pages/core/deb-debian/install-core.adoc

```diff
@@ -1,19 +1,10 @@
-.Install GnuPG and add OpenNMS repository GPG key
-[source, console]
-----
-sudo apt -y install curl gnupg ca-certificates
-curl -fsSL https://debian.opennms.org/OPENNMS-GPG-KEY | sudo gpg --dearmor -o /usr/share/keyrings/opennms.gpg
-----
-
 .Add and update apt repository
 [source, console]
 ----
-echo "deb [signed-by=/usr/share/keyrings/opennms.gpg] https://debian.opennms.org stable main" | sudo tee /etc/apt/sources.list.d/opennms.list
-sudo apt update
+curl -1sLf 'https://packages.opennms.com/public/stable/setup.deb.sh' | sudo -E bash
+curl -1sLf 'https://packages.opennms.com/public/common/setup.deb.sh' | sudo -E bash
```

Comment:
As both an experienced administrator and as an support engineer,  I curse with the fury of a thousand suns the person that decided  `curl whatever | sudo bash` was a good idea.  I implore you to not do this, especially for something as simple as _adding a debian repository_.  Tell the user what to do, don't try to mask it behind :star2: magic :star2: